com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -11,11 +11,50 @@
 using Unity.Multiplayer.Tools;
 #endif
 using Unity.Profiling;
+#if NETCODE_USE_ADDRESSABLES
+using UnityEngine.AddressableAssets;
+#endif
 using UnityEngine.SceneManagement;
 using Debug = UnityEngine.Debug;
 
 namespace Unity.Netcode
 {
+    public enum NetworkManagerState
```

Comment:
Feels like a natural thing to be in its own PR

Reply:
Had this discussion with Fatih. Could have been its own PR if I'd predicted the need for it and done it first... But addressables can't function without it, and it also has a dependency on Addressables unless I remove the LoadingAssets state and reimplement all the places the state gets changed. Splitting it at this point, to me, adds both time risk (unable to get this PR approved until that one's done and merged in) and code risk (the implementation of state on develop would look slightly different and that adds risk of partial bits of that ending up left in by accident after a merge)